src/components/dropdown-menu/DropdownMenuItemBase.tsx

```diff
@@ -127,7 +127,7 @@ const StyledIconCheck = styled(IconCheck)`
   font-size: var(--echoes-font-size-20);
 `;
 
-const CheckmarkPlaceholder = styled(StyledIconCheck)`
+const CheckmarkPlaceholder = styled(IconCheck)`
   visibility: hidden;
 `;
```

Comment:
Why this change?

Reply:
Nice catch, it should be reverted, I was testing if we really need the StyledIconCheck or not and we do.